coverage.py

```diff
@@ -341,6 +341,12 @@ def work(self, identifier):
         return work
 
 
+    def set_metadata(self, identifier, metadata, 
```

Comment:
I brought this method back because it's very popular. Many metadata sources only get Metadata, or they incorporate the CirculationData as metadata.circulation.